client/service/src/builder.rs

```diff
@@ -924,17 +924,20 @@ ServiceBuilder<
 			let is_validator = config.roles.is_authority();
 
 			let (import_stream, finality_stream) = (
-				client.import_notification_stream().map(|n| ChainEvent::NewBlock {
-					id: BlockId::Hash(n.hash),
-					header: n.header,
-					retracted: n.retracted,
-					is_new_best: n.is_new_best,
-				}),
+				futures::stream::select(client.import_notification_stream(), client.initial_sync_import_notification_stream())
```

Comment:
I think we should only use `all_blocks_notification_stream`, not both. There should be an extra parameter being in the stream to know if it's initial sync or not. That in turn should be forwarded to off-chain workers.

Reply:
Reverted that.